pkg/processor/build/builder.go

```diff
@@ -989,6 +1024,21 @@ func (b *Builder) getRuntimeNameByFileExtension(functionPath string) (string, er
 	}
 }
 
+func (b *Builder) getRuntimeFileExtensionByName(runtimeName string) (string, error) {
```

Comment:
This is not really needed and adds maintenance complexity (adding extensions per runtime). Let's remove it